### DISCUSSION TITLE
Feature request: Open index files in a directory instead of showing that directory

### DISCUSSION THREAD

**Discussion by Bazinga88** [DirectoryLister/DirectoryLister#948] [Ideas & Features]
Hi there. 

I hope this request is possible.

I use this tool on my webserver to list stuff on different drives in my network. 
At this moment when i click on the dir where webstuff is located i must click again on the index.php | index.php file. 
Is is possible to create an option so that when i click on the directory with an index.html | index.php file that the index automaticaly opens without a second click on the file?

This feature is listed in the readme: **Readme rendering allows exposing the contents of READMEs directly on the page**.
So i think it must possible. Same hook, other extension but i dont have the knowhow.

I think this freature wil make this tool much nicer ;-)
Thanks.

**Comment by Bazinga88** [comment]
Thanks for all the replies. I solved it myself. Have a nice day.

**Comment by timangus** [comment]
Could you explain how you solved this? Thanks.
